code/core/src/androidTest/java/com/adobe/marketing/mobile/services/ui/message/views/MessageScreenTests.kt

```diff
@@ -1549,6 +1674,83 @@ class MessageScreenTests {
+            .content(HTML_TEXT_SAMPLE)
+            .height(heightPercent)
+            .width(widthPercent)
+            .maxWidth(maxWidthPx)
+            .verticalAlignment(InAppMessageSettings.MessageAlignment.BOTTOM)
+            .verticalInset(offsetPercent)
+            .horizontalAlignment(InAppMessageSettings.MessageAlignment.RIGHT)
+            .horizontalInset(-offsetPercent)
+            .build()
+
+        var contentViewHeightDp = 0.dp
+        var messageContentHeightDp = 0.dp
+        var messageContentWidthDp = 0.dp
+        composeTestRule.setContent { // setting our composable as content for test
+            val activity = LocalContext.current as Activity
+            val contentView = activity.findViewById<View>(android.R.id.content)
+            contentViewHeightDp = with(LocalDensity.current) { contentView.height.toDp() }
+            messageContentHeightDp = ((contentViewHeightDp * settings.height) / 100)
+            messageContentWidthDp = with(LocalDensity.current) { maxWidthPx.toDp() }
+
+            MessageScreen(
+                presentationStateManager = presentationStateManager,
+                inAppMessageSettings = settings,
+                onCreated = { onCreatedCalled = true },
+                onDisposed = { onDisposedCalled = true },
+                onGestureDetected = { gesture -> detectedGestures.add(gesture) },
+                onBackPressed = { onBackPressed = true }
+            )
+        }
+
+        composeTestRule.waitForIdle()
+        composeTestRule.onNodeWithTag(MessageTestTags.MESSAGE_FRAME).assertDoesNotExist()
+
+        // Change the state of the presentation state manager to shown to display the message
+        presentationStateManager.onShown()
+        composeTestRule.waitForIdle()
+        validateMessageAppeared(
+            composeTestRule = composeTestRule,
+            withBackdrop = false,
+            clipped = true
+        )
+
+        val rootBounds = composeTestRule.onRoot().getBoundsInRoot()
+        val frameBounds =
+            composeTestRule.onNodeWithTag(MessageTestTags.MESSAGE_FRAME).getUnclippedBoundsInRoot()
+        val contentBounds = composeTestRule.onNodeWithTag(MessageTestTags.MESSAGE_CONTENT)
+            .getUnclippedBoundsInRoot()
+        val heightOffsetDp = messageContentHeightDp * offsetPercent.toFloat() / 100f
+        val widthOffsetDp = messageContentWidthDp * offsetPercent.toFloat() / 100f
+
+        // Frame top and bottom  bounds are offset upwards by the inset from the bottom
+        // left and right bounds are offset rightwards by the inset value from the right
+        validateBounds(
+            frameBounds,
+            rootBounds.top - heightOffsetDp,
+            rootBounds.bottom - heightOffsetDp,
+            rootBounds.left + widthOffsetDp,
+            rootBounds.right + widthOffsetDp
+        )
+
+        // Content is bottom aligned vertically and offset upwards and centered horizontally and takes 80% of screen width
```

Comment:
Since the maxWidth of 50px is reasonably less than 80% of screen width, I think the `messageContentWidthDp` will be set to `maxWidthPx.toDp()` which the assertion validates. Should this comment be `....ad horizontally and takes specified max width`?

Reply:
Copied over from another test, fixed it now